FlowerinoPlugin/src/org/flowerplatform/flowerino_plugin/FlowerPlatformPlugin.java

```diff
@@ -125,13 +143,20 @@ protected void showLibraryManager(String resourceNodeUri, boolean showDialogOnly
 	}
 
 	public static AbstractLibraryInstallerWrapper libraryInstallerWrapper;
-	
+
+	private static FlowerPlatformPlugin INSTANCE;

```

Comment:
as prefera sa nu facem treaba asta. Adica fiecare clasa sa aiba o proprietate, si sa i se dea la creere.
E un pic problematic pentru comenzi. Insa, ar trebui sa introducem si notiunea de factory. I se da controlul pentru crearea unei clase. Si daca clasa is IFlowerPlatformPluginAware, atunci .setFlPlPlugin()